example_data/somatic_exome.yaml

```diff
@@ -75,3 +75,4 @@ interval_list:
   path: exome_workflow/chr17_test_genes.interval_list
 mutect_scatter_count: 2
 mutect_artifact_detection_mode: false
+somalier_vcf: "exome_workflow/chr17_vcf_file_for_somalier.vcf"
```

Comment:
`chr17_vcf_file_for_somalier.vcf` should be added to the `exome_workflow.tgz` file instead of being added directly to the repository if this is the chosen path.

Reply:
Yes i did it and added the new exome_workflow.tgz

Reply:
I don't see the new one in this PR.  Which commit adds the new version?